result/uds.yaml

```diff
@@ -0,0 +1,101 @@
+components:
+  schemas:
+    User.Defined.Metrics.Request:
```

Comment:
The use of the term "User Defined Stats" seems  a bit odd, given that you've defined a large number of built-in stats, including an option of "custom". "custom" and "user-defined" are essentially synonymous, so perhaps we should look for a better name.

Reply:
Currently we are using this 
```go 
gosnappi.NewMetricsRequest().UserDefined()
```
Here are some alternatives I was thinking

```go 
gosnappi.NewMetricsRequest().Customized()
```
```go 
gosnappi.NewMetricsRequest().Tailored()
```
```go 
gosnappi.NewMetricsRequest().Targeted()
```

Let me know if you have some suggestion or you would like one of the above.